frontend/src/hocs/withHostCommunication/types.ts

```diff
@@ -108,6 +108,10 @@ export type IHostToGuestMessage = {
       type: "UPDATE_HASH"
       hash: string
     }
+  | {
+      type: "SET_CUSTOM_THEME_CONFIG"
+      theme: ICustomThemeConfig
```

Comment:
Not sure why this doesn't trigger an error, but need to update this payload to be called `themeInfo` consistent with every other place updated in [this commit](https://github.com/streamlit/streamlit/pull/6393/commits/863d59f9d815cd746a9c375fae833278bda1900b)

Reply:
Fixed in [0486457](https://github.com/streamlit/streamlit/pull/6393/commits/0486457167675fc033f22a58ba70795f6a7c1a7a)!